Sharpmake.Generators/VisualStudio/ProjectOptionsGenerator.cs

```diff
@@ -695,6 +695,16 @@ private void GenerateCompilerOptions(IGenerationContext context, ProjectOptionsG
             Options.Option(Options.Vc.Compiler.KeepComment.Enable, () => { context.Options["KeepComments"] = "true"; context.CommandLineOptions["KeepComments"] = "/C"; })
             );
 
+            //Options.Vc.Compiler.UseStandardConformingPreprocessor.    See: https://learn.microsoft.com/en-us/cpp/build/reference/zc-preprocessor?view=msvc-170
+            //    Disable                                 /Zc:preprocessor-
+            //    Enable                                  /Zc:preprocessor
+            context.SelectOption
```

Comment:
Please add support for generating the option in vcxproj

![image](https://github.com/user-attachments/assets/c90f22c8-79e0-4805-bb0d-fc4621bd9cf0)

Reply:
**Sharpmake.Options.Vc.Compiler.UseStandardConformingPreprocessor** now sets the "UseStandardPreprocessor" in the vcxproj.